cmd/libs/go2idl/client-gen/output/deployment.go

```diff
@@ -0,0 +1,139 @@
+/*
```

Comment:
I think that generating it to:
cmd/libs/...
doesn't make much sense.
I think that we should just generate it into pkg/client/unversioned and replace the existing file with it.
WDYT?